XenAdmin/Controls/MultipleDvdIsoList.cs

```diff
@@ -51,6 +51,8 @@ public partial class MultipleDvdIsoList : UserControl
         public MultipleDvdIsoList()
         {
             InitializeComponent();
+            labelSingleDvd.ForeColor = Program.HeaderGradientForeColor;

```

Comment:
Aah, right, there it's placed on white and not on the teal bit, I missed that - I'll revert the change, thanks for spotting it.